src/message_constants.ts

```diff
@@ -40,7 +40,7 @@ export function CLARIFY_STATE(): string {
 export function WELCOME_VOTER(): string {
   switch (process.env.CLIENT_ORGANIZATION) {
     case 'VOTE_AMERICA':
-      return 'Welcome to VoteAmerica! We are excited to help you vote and can connect you to a trained volunteer to help answer your questions.\n\nReply HELPLINE to be connected to a volunteer. Reply STOP to unsubscribe from VoteAmerica texts. (Msg & data rates may apply).';
+      return "Welcome to VoteAmerica! We're excited to help you vote.\n\nReply HELPLINE to speak to a trained volunteer. Reply STOP to unsubscribe. (Msg&data rates may apply)";
```

Comment:
This is great. I'm a bit concerned that a voter might think "speak" implies a phone call. How about:

Welcome to VoteAmerica! We're excited to help you vote.\n\nReply HELPLINE to connect with a volunteer. Reply STOP to unsubscribe. (Msg & data rates may apply).

Reply:
updated. also modified the welcome + state message to fit in 160 chars